extensions-core/kafka-indexing-service/src/main/java/org/apache/druid/indexing/kafka/KafkaRecordSupplier.java

```diff
@@ -200,28 +213,32 @@ public static void addConsumerPropertiesFromConfig(
       }
     }
   }
-  
+
   private Deserializer getKafkaDeserializer(Properties properties, String kafkaConfigKey)
   {
     Deserializer deserializerObject;
     try {
-      Class deserializerClass = Class.forName(properties.getProperty(kafkaConfigKey, ByteArrayDeserializer.class.getTypeName()));
+      Class deserializerClass = Class.forName(properties.getProperty(
+          kafkaConfigKey,
+          ByteArrayDeserializer.class.getTypeName()
+      ));
       Method deserializerMethod = deserializerClass.getMethod("deserialize", String.class, byte[].class);
-      
+
       Type deserializerReturnType = deserializerMethod.getGenericReturnType();
-      
+
       if (deserializerReturnType == byte[].class) {
         deserializerObject = (Deserializer) deserializerClass.getConstructor().newInstance();
       } else {
-        throw new IllegalArgumentException("Kafka deserializers must return a byte array (byte[]), " + deserializerClass.getName() + " returns " + deserializerReturnType.getTypeName());
+        throw new IllegalArgumentException("Kafka deserializers must return a byte array (byte[]), " + deserializerClass
```

Comment:
It's better to but the line break at a higher syntactic level: https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break

i.e., instead of:
```
"Kafka deserializers must return a byte array (byte[]), " + deserializerClass
.getName() + "returns "
```

something like:
```
"Kafka deserializers must return a byte array (byte[]), " +
deserializerClass.getName() + "returns "
```